tealium/__init__.py

```diff
@@ -45,16 +45,18 @@ class Tealium(object):
 
     '''
         Returns an instance of a Tealium object, path is optional.
+
+        "path" is used for persistence
     '''
```

Comment:
Let's fully describe all the arguments here while we have the chance.  Path description could be more detailed, something like ".. path to alternate persistence storage location. i.e. /usr/bin/local"